python/lsst/afw/geom/ellipses/_ellipses.cc

```diff
@@ -0,0 +1,60 @@
+/*
+ * LSST Data Management System
+ * Copyright 2008-2021 AURA/LSST.
+ *
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <https://www.lsstcorp.org/LegalNotices/>.
+ */
+
+#include "pybind11/pybind11.h"
+#include "lsst/utils/python.h"
+
+namespace lsst {
+namespace afw {
+namespace geom {
+namespace ellipses {
+void wrapEllipse(lsst::utils::python::WrapperCollection &);
+void wrapPixelRegion(lsst::utils::python::WrapperCollection &);
+void wrapBaseCore(lsst::utils::python::WrapperCollection &);
+void wrapAxes(lsst::utils::python::WrapperCollection &);
+void wrapRadii(lsst::utils::python::WrapperCollection &);
+void wrapEllipticityBase(lsst::utils::python::WrapperCollection &);
+void wrapDistortion(lsst::utils::python::WrapperCollection &);
+void wrapConformalShear(lsst::utils::python::WrapperCollection &);
+void wrapReducedShear(lsst::utils::python::WrapperCollection &);
+void wrapQuadrupole(lsst::utils::python::WrapperCollection &);
+void wrapSeparable(lsst::utils::python::WrapperCollection &);
+
+PYBIND11_MODULE(_ellipses, mod) {
+    lsst::utils::python::WrapperCollection wrappers(mod, "lsst.afw.geom.ellipses");
```

Comment:
Unless I missed it in other source files for this module, should have a signature dependency on `lsst.geom`.